fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/common/CheckPointAlgorithm.java

```diff
@@ -30,16 +30,12 @@
     @Inject
     StepContext stepCtx;
 
-    /**
-     * The file resources number limit when exporting to multiple COS files.
-     */
+    // The max number of resources to store in a single COS object
```

Comment:
I like the description change, but I prefer the javadoc-style comment because then we get nice hover behavior in eclipse; useful when working in the class if you forget specifics about the field.

Reply:
It's a pity they're not picked up with that style. I find the Javadoc style reduces code-density too much for member variables.

Reply:
yeah, could go with `/** comment here */` to get it on one line, but I agree that just doesn't look quite as satisfying